lib/cypress_rails/middleware.rb

```diff
@@ -42,7 +42,15 @@ def setup!
 
     def execute_script!(request)
       body = JSON.parse(request.body.read)
-      CypressRails.scripts(body.fetch("name")).call
+      script = CypressRails.scripts(body.fetch("name"))
+
+      params = body.fetch("params", {})
+
+      if params.any?
```

Comment:
Would it works if you run `script.call(params)` and ignore `script.call` version ?